cfme/ansible_tower/explorer.py

```diff
@@ -85,7 +86,8 @@ class TowerExplorerJobTemplatesAllView(TowerExplorerView):
     def is_displayed(self):
         return (
             self.in_tower_explorer and
-            self.title.text == 'All Ansible Tower Job Templates' and
+            self.title.text == VersionPick({Version.lowest(): 'All Ansible Tower Job Templates',
+                                       '5.10': 'All Ansible Tower Templates'}) and
```

Comment:
**REQUIRED**
```python
self.title.text == VersionPicker({
    Version.lowest(): 'All Ansible Tower Job Templates',
    '5.10': 'All Ansible Tower Templates'}).pick(self.appliance.version)
```

Reply:
Added versionpicker: https://github.com/ManageIQ/integration_tests/pull/7785/files#diff-ca2caa6aac1e64d4303659fee714d350R89 But it fails on that it hasn't self.appliance.

Reply:
Try
```python
self.title.text == VersionPicker({
    Version.lowest(): 'All Ansible Tower Job Templates',
    '5.10': 'All Ansible Tower Templates'}).pick(self.browser.product_version)